.circleci/config.yml

```diff
@@ -72,6 +72,55 @@ jobs:
       - run: |
           bash <(curl -s https://codecov.io/bash)
 
+  test-hsm:
+    docker:
+      - image: cimg/go:1.16-node
+        environment:
+          - HSM_ENABLED=true
+          - HSM_LIBRARY=/usr/lib/softhsm/libsofthsm2.so
+          - HSM_TOKEN_LABEL=hydra
+          - HSM_PIN=1234
+          - TEST_DATABASE_POSTGRESQL=postgres://test:test@localhost:5432/postgres?sslmode=disable
+          - TEST_DATABASE_MYSQL=mysql://root:test@(localhost:3306)/mysql?multiStatements=true&parseTime=true
+          - TEST_DATABASE_COCKROACHDB=cockroach://root@localhost:26257/defaultdb?sslmode=disable
+      - image: postgres:9.6
+        environment:
+          - POSTGRES_USER=test
+          - POSTGRES_PASSWORD=test
+          - POSTGRES_DB=postgres
+      - image: mysql:8.0
+        environment:
+          - MYSQL_ROOT_PASSWORD=test
+      - image: cockroachdb/cockroach:v20.2.5
+        command: start-single-node --insecure
+    steps:
+      - checkout
+      - setup_remote_docker
+
+      - go/load-cache:
+          key: ory-hydra-go-mod-v1
+      - go/mod-download
+      - go/save-cache:
+          key: ory-hydra-go-mod-v1
+
+      - nancy/install
+      - nancy/check
+      - prettier/install
+      - prettier/check
+      - prettier/install:
+          dir: docs/
+      - prettier/check:
+          dir: docs/
+      - goreleaser/install
+      - goreleaser/check
+      - golangci/install
+      - golangci/lint
```

Comment:
I guess some of these steps are duplicated in test job can be removed here

Reply:
Yes, we only need the things for HSM here, no need for goreleaser checks etc!